src/server/acl/acl_family_test.cc

```diff
@@ -0,0 +1,158 @@
+// Copyright 2022, DragonflyDB authors.  All rights reserved.
+// See LICENSE for licensing terms.
+//
+
+#include "server/acl/acl_family.h"
+
+#include "absl/strings/str_cat.h"
+#include "base/gtest.h"
+#include "base/logging.h"
+#include "facade/facade_test.h"
+#include "server/acl/acl_commands_def.h"
+#include "server/command_registry.h"
+#include "server/test_utils.h"
+
+using namespace testing;
+using namespace std;
+using namespace util;
+using namespace boost;
```

Comment:
What's that for 🤨

Reply:
Ah I always fall for your auto merge 😆

Reply:
@dranikpg it's a trap! 

I will remove this, hold!